### DISCUSSION TITLE
mivlvus v2.0.2 can't support GPU mode?

### DISCUSSION THREAD

**Discussion by PanternBao** [milvus-io/milvus#21767] [Q&A and General discussion]
As far as I know, faiss supports gpu index. 
Knowhere partly is based on faiss. 
Why mivlus  doesn't support gpu mode, how the milvus team thinks about it?

**Comment by yhmo** [comment]
No, v2.0/2.1/2.2 can't support GPU.
In our plan, the v2.3 can support GPU to build index: https://wiki.lfaidata.foundation/display/MIL/Feature+plans+for+Milvus+2.3+and+timeline


The milvus v2.x is mainly targeted to massive vectors(even billions). GPU memory is not large enough to hold massive vectors, so this feature is not a high priority in our plan.
